### PR TITLE
add go base dockerfile

### PR DESCRIPTION
**Pull request by samaross**

### CODE REVIEW

**Comment by fluffywaffles** [comment]
With the refactor that just landed, there's no longer a common.sh

File: base/Dockerfile.go [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM golang:1.13.8-alpine
+
+LABEL maintainer="core-tech@better.com"
+
+COPY scripts/common.sh common.sh
+RUN apk update && apk upgrade && chmod +x common.sh && ./common.sh
```
Comment (fluffywaffles) [inline base/Dockerfile.go]: This now needs to look like the similar sequences from `base/Dockerfile.node` and `base/Dockerfile.python`:

```
ARG BASE_APK_DEPENDENCIES

# Install the base requirements
COPY                                         \
  scripts/common/install-rds-certificates.sh \
  scripts/common/install-sops.sh             \
  /tmp/scripts/common/
RUN :                                                         \
  && apk update                                               \
  && apk upgrade                                              \
  && apk add --no-cache                                       \
    # Base dependencies                                       \
    ${BASE_APK_DEPENDENCIES}                                  \
  # Run common install scripts                                \
  && chmod +x /tmp/scripts/common/install-rds-certificates.sh \
  && chmod +x /tmp/scripts/common/install-sops.sh             \
  && /tmp/scripts/common/install-rds-certificates.sh          \
  && /tmp/scripts/common/install-sops.sh                      \
  ;
```

**Comment by samaross** [comment]
thanks @fluffywaffles !

**Comment by fluffywaffles** [comment]
Godspeed! You should just be able to `make build-base-go` and then `make
release-base-go` to check it locally and then push it to Docker Hub

**Pull request merged**